### ISSUE TITLE
Report the user that is giving a grade

### ISSUE THREAD

**Issue by rjglasse** [slarse/repobee-csvgrades#12]
Motivation: It can be problematic to map the teaching assistant to student at the start of the course, and this can then change in the future. It would be helpful if the plugin could accept an argument to also report the last person that gave a grade. If the history of TA is required then the .json file can be searched.

Example:
Teacher `glassey` reports grades on student `somest` then uses the plugin to report grades:

`$ repobee record-grades --mn task-{1..2} --s somest --teacher`

which yields:

```
teacher,        student, username, task-1, task-2
glassey, "some student",   somest,      P,      F
```

There might be a better term than teacher, but you get the idea

**Comment by slarse** [comment]
@rjglasse I very much forgot about this :)

In any case, the plugin all ready resolves which teacher/TA reports which grade, which is then printed in the edit message, so in principle this is not problematic.

I'm not entirely sure about the usability aspect of it, though. It doesn't make a whole lot of sense for this to be automatic, as then for example reporting tasks 6 and 7 at one point, and then after that reporting tasks 1 and 2 for stragglers, may cause unexpected changes in the teacher/TA assignment.

Perhaps one way to do it would be to have an `--update-teacher` option (is that the point of your `--teacher` option?), which updates the teacher field with the latest reported grade based on the information available in the hook results file (we need to come up with a better name for that thing...). It would possibly make sense to somehow enforce that data for all tasks is present, such that you don't accidentally update teacher's based on e.g. only the first task.

**Comment by rjglasse** [comment]
Yeah, on reflection #13 covers the main problem area of discovering missing grades in a more useful fashion as a teacher would still have to scan the CSV file for gaps :)
